lib/PuppeteerSharp.Tests/PageTests/CloseTests.cs

```diff
@@ -17,7 +17,7 @@ public async Task ShouldRejectAllPromisesWhenPageIsClosed()
             // Put into a var to avoid warning
             var t = Page.CloseAsync();
 
-            var exception = await Assert.ThrowsAsync<TargetClosedException>(async () => await neverResolves);
+            var exception = await Assert.ThrowsAnyAsync<PuppeteerException>(async () => await neverResolves);
```

Comment:
this should use `ThrowsAsync` instead

Reply:
I changted this to `Any` because this changed from `TargetClosedException` to `EvaluationException`(which has `TargetClosedException` as inner exception).
So here I changed to any because we don't care if which exception is as long as we throw something.

What do you think?

Reply:
since the test tests specifically that the method throws an exception I think this test should care about the exception type